pystac_monty/sources/pdc.py

```diff
@@ -39,7 +43,7 @@ class PDCTransformer(MontyDataTransformer):
     """Transform the source data into the STAC items"""
 
     hazard_profiles = MontyHazardProfiles()
-    source_name = 'pdc'
+    source_name = "pdc"
```

Comment:
The file loading should be moved out of the transfomer class and moved into the data source class